File: Entidades/DetalleEN.vb
Public Class DetalleEN
    Private _codnot As Integer
    Public Property CodNot() As Integer
        Get
            Return _codnot
        End Get
        Set(ByVal value As Integer)
            _codnot = value
        End Set
    End Property

    Private _codprod As Integer
    Public Property CodProd() As Integer
        Get
            Return _codprod
        End Get
        Set(ByVal value As Integer)
            _codprod = value
        End Set
    End Property


    Private _cantidad As Integer
    Public Property Cantidad() As Integer
        Get
            Return _cantidad
        End Get
        Set(ByVal value As Integer)
            _cantidad = value
        End Set
    End Property

    Private _precio As String
    Public Property Precio() As String
        Get
            Return _precio
        End Get
        Set(ByVal value As String)
            _precio = value
        End Set
    End Property


    Private _nombreproducto As String
    Public Property NombreProducto() As String
        Get
            Return _nombreproducto
        End Get
        Set(ByVal value As String)
            _nombreproducto = value
        End Set
    End Property


End Class

File: Entidades/DetalleEN.cs

namespace Entidades
{
    public class DetalleEN
    {
        private int _codnot;

        public int CodNot
        {
            get
            {
                return _codnot;
            }

            set
            {
                _codnot = value;
            }
        }

        private int _codprod;

        public int CodProd
        {
            get
            {
                return _codprod;
            }

            set
            {
                _codprod = value;
            }
        }

        private int _cantidad;

        public int Cantidad
        {
            get
            {
                return _cantidad;
            }

            set
            {
                _cantidad = value;
            }
        }

        private string _precio;

        public string Precio
        {
            get
            {
                return _precio;
            }

            set
            {
                _precio = value;
            }
        }

        private string _nombreproducto;

        public string NombreProducto
        {
            get
            {
                return _nombreproducto;
            }

            set
            {
                _nombreproducto = value;
            }
        }
    }
}